cli.js

```diff
@@ -96,12 +103,11 @@ const argv = require('yargs')
           .positional('components', { describe: 'Delimited by colon components, i.e. maven:boost:bootstrap', default: '', type: 'string' })
           .option('d', { describe: 'depth level', alias: 'depth', default: -1, type: 'number' })
       },
-      (argv) => {
-        const log = logger.create(argv.verbose);
-        require('./src/appl').create(log, cwd, __dirname, argv.presetsDest)
-          .resolve(argv.components).forEach( (component) => {
+      async (argv) => {
+        const {l, f, t, a} = await scope(argv.verbose, argv.presetsDest);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "t" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.65.230.204:9000/coding_rules#rule_key=javascript%3AS1854)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "f" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.65.230.204:9000/coding_rules#rule_key=javascript%3AS1854)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "l" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.65.230.204:9000/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'l' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.65.230.204:9000/coding_rules#rule_key=javascript%3AUnusedVariable)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'f' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.65.230.204:9000/coding_rules#rule_key=javascript%3AUnusedVariable)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 't' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://159.65.230.204:9000/coding_rules#rule_key=javascript%3AUnusedVariable)